src/LCT.APICommunications/JfrogAqlApiCommunication.cs

```diff
@@ -80,9 +88,8 @@ public async Task<HttpResponseMessage> GetPackageInfo(string repoName, string pa
         private static HttpClient GetHttpClient(ArtifactoryCredentials credentials)
         {
             HttpClient httpClient = new HttpClient();
-
-            httpClient.DefaultRequestHeaders.Add(ApiConstant.JFrog_API_Header, credentials.ApiKey);
-            httpClient.DefaultRequestHeaders.Add(ApiConstant.Email, credentials.Email);
```

Comment:
If email is no longer required, we need to also remove it from the app settings also

Reply:
Added In Usage Document as optional.